docs-site/docs/docs/server-configuration.md

```diff
@@ -1,4 +1,9 @@
 To change any of the default configuration, you'll need to copy the included [`config-template.yml`](https://github.com/polynote/polynote/blob/master/config-template.yml)
 file to `config.yml`, and uncomment the sections you'd like to change. Check out the template itself for more information.
 
-### Storage
\ No newline at end of file
+### Storage
```

Comment:
In this case I believe that `Storage` was meant to refer to https://github.com/polynote/polynote/blob/master/config-template.yml#L36-L49

Reply:
Gotcha - I'll leave this for later as I wanted to have a broader conversation about surfacing the configuration file sometime soon.